syntax = "proto3";

package smartnvr20;

service FrameHandlerService {
    rpc SendHeartbeat (HeartbeatRequest) returns (HeartbeatResponse) {}
}

message HeartbeatRequest {
    string client_id = 1;
    int64 timestamp = 2;
}

message HeartbeatResponse {
    bool success = 1;
    string message = 2;
}